### ISSUE TITLE
creating a user with no @ sign possible but can't login anymore

### ISSUE THREAD

**Issue by ermethic** [jlesage/docker-nginx-proxy-manager#121]
Hello,

The initial setup went fine, but I created an admin user with just a nickname (no email address format), and deleted the default admin. System didn't complain about a wrong username format. After that I configured all my proxies with no issue.
After logging out I can't login anymore because the username is not in an email address format, and even if I reset password via docker exec, still can't login (same error) - "please include an @ in the email address".
I'm locked out now, is there anything I can do besides reinstalling and loosing all my config?

thanks.

**Comment by jlesage** [comment]
You can try this to change the username back to an email:  first, login to the container:
```
docker exec -ti <name of the container> sh
```
Then execute:
```
echo "UPDATE user SET user.email = 'new@example.com' WHERE user.email = 'old@example.com'" | mysql nginxproxymanager
```

Make sure to adjust `old@example.com` and `new@example.com`.

**Comment by ermethic** [comment]
not working, doesn't find mysql...

`sh: mysql: not found`

**Comment by jlesage** [comment]
Which version of the image are you using ?

**Comment by ermethic** [comment]
It's the latest docker version under unraid. I force-updated it and still the same.

**Comment by jlesage** [comment]
Ok, the latest image doesn't have the same database.  Try instead:
```
echo "UPDATE user SET email = 'new@example.com' WHERE email = 'old@example.com'" | sqlite3 /config/database.sqlite
```

**Comment by ermethic** [comment]
No error this time, but still `No relevant user found` when i try to login with the new email.
Also tried to restart the docker afterwards, and tried a password reset, but the issue stands.

**Comment by jlesage** [comment]
Has your user been updated correctly?
```
echo "SELECT * from user" | sqlite3 /config/database.sqlite
```

**Comment by ermethic** [comment]
no, sorry, i gave up and reinstalled and reconfigured the whole thing...
